### PR TITLE
return hello x many times

### PR DESCRIPTION
**Pull request by kevinYCKim33**

### CODE REVIEW

File: curriculum/js1/3.js [hunk 1]
```diff
@@ -5,8 +5,12 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a,b) => {
+  var arr = [];
+  for (let i = 0; i < a; i++) {
```
Comment (songz) [inline curriculum/js1/3.js]: don't use for loops.

Comment (seemcat) [reply curriculum/js1/3.js]: turn this into a recursive function

**Review by seemcat — CHANGES_REQUESTED** [review]
turn into a recursive function

File: curriculum/js1/3.js [hunk 2]
```diff
@@ -5,10 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
-};
+const solution = (a,result='') => {
+  if (a === 0) return result;
+  return solution(a-1, result += 'hello')
```
Comment (songz) [inline curriculum/js1/3.js]: `result + 'hello'`

**Review by songz — APPROVED** [review]

**Pull request merged**